### PR TITLE
Move edit taxonomy/topics page to the GOV.UK Design System

### PR DESCRIPTION
**Pull request by DilwoarH**
## What

Move edit taxonomy/topics page to the GOV.UK Design System

## Why

This is part of the work to move Whitehall over to the GOVUK design system

https://trello.com/c/SgpMGp4I/611-move-edit-taxonomy-topics-page-to-the-govuk-design-system

## Before

screenshot-whitehall-admin dev gov uk-2022 08 17-18_11_59

## After

screenshot-whitehall-admin dev gov uk-2022 08 18-20_23_07

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

File: app/views/admin/edition_tags/edit.html.erb [hunk 1]
```diff
@@ -1,76 +1,83 @@
...
+      <% end %>
 
       <%= form.hidden_field "invisible_taxons", value: @tag_form.invisible_taxons.join(",") %>
 
-      <p>
+      <p class="govuk-body">
         <%= link_to "Suggest a new tag",
           "#{Whitehall.support_url}/taxonomy_new_topic_request/new",
-          class: "feedback-link"
+          class: "feedback-link govuk-link"
```
Comment (davidtrussler) [inline app/views/admin/edition_tags/edit.html.erb:47]: Can we can remove the class `feedback-link` now - it doesn't seem to do anything?

File: app/views/admin/edition_tags/edit.html.erb [hunk 2]
```diff
@@ -1,76 +1,83 @@
...
         %>
         or
         <%= link_to "Suggest a change to a tag",
           "#{Whitehall.support_url}/taxonomy_change_topic_request/new",
-          class: "feedback-link"
+          class: "feedback-link govuk-link"
```
Comment (davidtrussler) [inline app/views/admin/edition_tags/edit.html.erb:52]: Same comment as line 47 on `feedback-link` class.

File: app/views/admin/edition_tags/edit.html.erb [hunk 3]
```diff
@@ -1,76 +1,83 @@
...
-                    track_category: 'taxonSelection',
-                    track_label: request.path
-                }) %>
-          <% end %>
-          <span class="or_cancel">
-            or
-            <%= link_to('cancel', admin_edition_path(@edition)) %>
-          </span>
-        </div>
+        <%= link_to('Cancel', admin_edition_path(@edition), class: "govuk-link") %>
```
Comment (davidtrussler) [inline app/views/admin/edition_tags/edit.html.erb:80]: Should this be replaced with the `Back` link to match the new design?

File: app/views/admin/edition_tags/edit.html.erb (hunk 1 above)
Comment (DilwoarH) [reply app/views/admin/edition_tags/edit.html.erb]: done

File: app/controllers/admin/base_controller.rb [hunk 4]
```diff
@@ -52,6 +52,10 @@ def forbid_editing_of_locked_documents
     end
   end
 
+  def preview_design_system_user?
+    current_user.has_permission? "Preview Design System"
```
Comment (davidgisbey) [inline app/controllers/admin/base_controller.rb:56]: I think this needs to be "Preview design system". There's a convention in whitehall to lowercase permission strings (other than the first word) which i followed

Comment (DilwoarH) [reply app/controllers/admin/base_controller.rb]: done

**Review by davidgisbey — APPROVED** [review]
Thanks for sorting the feature spec. Gave it a good manual test and it's working nicely

**Review by davidtrussler — APPROVED** [review]
LGTM 👍

**Pull request merged**